app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
```

Comment:
I don't think we need a `UsersController` since this should be taken care of by Devise